README.md

```diff
@@ -1,6 +1,104 @@
-# Running
-- install postgresql (including the headers, `postgresql-server-dev` on ubuntu)
-- create a user and a database
-- run the following on your test database: `CREATE SEQUENCE users_usr_id_seq;`
-- import the users table
-- update the database and credentials in `config/database.yml`
+# PéK-Next
+
+**The administration system for Schönherz Zoltán Student Hostel and [SVIE](http://svie.hu/)**
+
+[![Build Status](https://travis-ci.org/kir-dev/pek-next.png?branch=master)](https://travis-ci.org/kir-dev/pek-next)
+[![Code Climate](https://codeclimate.com/github/kir-dev/pek-next.png)](https://codeclimate.com/github/kir-dev/pek-next)
+[![Dependency Status](https://gemnasium.com/kir-dev/pek-next.png)](https://gemnasium.com/kir-dev/pek-next)
+
+## Running
+
+**To run PéK you will need the following:**
+
+###### Postgresql packages
+
+```bash
+$ sudo apt-get install postgresql libpq-dev nodejs
+```
+
+###### Ruby 2.4.2 environment
+
+```bash
+$ sudo apt-get install rbenv
+$ mkdir -p "$(rbenv root)"/plugins
+$ git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
+$ echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+$ echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+$ source ~/.bashrc
```

Comment:
i appreciate the effort but keep in mind that rbenv might change. Do you want to maintain their installation manual here? What do you think about just referring their readme? Hopefully everyone is smart enough to follow the instructions there :D

Reply:
The rbenv install had quite a few challenges, so I think it would be optimal to keep a simplified version here. This way future developers with less Linux experience won't get stuck trying to figure out what the error messages mean.
Maybe adding a 
"The guide is tested for rbenv 1.1.1, for newer versions please refer to [rbenv github page](https://github.com/rbenv/rbenv)"
line could help with possible future changes.